docs/docs/user-guide/permissions.md

```diff
@@ -0,0 +1,108 @@
+
+- **Add Collaborators**: In the **Collaborators** section, you can add new users as collaborators to the database. When adding a collaborator, you assign them one of the roles that have been configured in Mathesar.
+
+- **Remove Collaborators**: You can remove collaborators if they no longer require access to the database. The Mathesar user will still exist and you can add them back as a collaborator at a later time.
+
+### Server Roles
+
+The **Server Roles** section, found under the **Settings** tab, shows the roles available on the server for the current database.
+
+These roles will be displayed for all databases on the server. Also, any changes made to a server role will be reflected in all databases on the server.
+
+- **Create Roles**: You can create new server-level roles from this section. You can configure these roles in two ways:
+    1. With login capability and a password, which you can assign to collaborators.
+    2. Without login capability, to be used as a parent role exclusively. You cannot assign these roles to collaborators directly.
+- **Define Child Roles**: PostgreSQL has a mechanism for [Role Membership](https://www.postgresql.org/docs/current/role-membership.html) wherein any role can be "granted" to any other role to form simple hierarchies or complex graph-based inheritance structures. For any role you've configured within Mathesar, you can use Mathesar to grant the role to other "child roles".
+- **Drop Roles**: You can drop server-level roles that are no longer needed. This action removes the role from the server and all databases where it has been assigned. Exercise caution when dropping roles, as it may affect existing permissions and user access across multiple databases.
+
+!!! note
+    Server roles, once added, must be configured in Mathesar under the **Role Configuration** section before they can be assigned to collaborators.
+
+---
+
+## Permissions Management at Different Levels
+
+Mathesar manages permissions at three levels:
+
+1. **Database Level:** Managed from the database page.
+2. **Schema Level:** Managed from the schema page.
+3. **Table Level:** Managed from the inspector panel for each table.
+
+### Ownership of Objects
+
+Owners of each object (database, schema, or table) can set permissions for other users. Ownership is determined by the user's assigned collaborator role in the database.
+
+When a new object (database, schema, or table) is created, the owner of the object is the collaborator who created it. This automatic ownership assignment allows the creator to have immediate and full control over the objects they create, including the ability to manage permissions and transfer ownership if needed.
+
+### Database Permissions
+
+Database permissions control access and actions at the database level.
+
+- **Owner**: Each database has an owner who has full control over the database, including managing permissions and transferring ownership.
```

Comment:
> full control over the database

We shouldn't be calling this full control. It provides access only over the database, not on the objects inside the database. We need to make this clear.

Reply:
Changed all instances.